pkg/api/norman/customization/clusterregistrationtokens/formatter.go

```diff
@@ -101,7 +101,9 @@ func AgentEnvVars(cluster *v3.Cluster) string {
 	var agentEnvVars []string
 	if cluster != nil {
 		for _, envVar := range cluster.Spec.AgentEnvVars {
-			agentEnvVars = append(agentEnvVars, fmt.Sprintf("-e %s=%s", envVar.Name, envVar.Value))
+			if envVar.Name != "" && envVar.Value != "" {
```

Comment:
Can you add a comment we are explicitly dropping valueFrom and link to the issue for the docs as it being currently unsupported?

Reply:
✅